src/PartielsPy/export_configs/_export_config.py

```diff
@@ -0,0 +1,34 @@
+"""An abstract class for export configuration"""
+
+
+class ExportConfig:
+    """Export configuration class
+
+    This class is used to configure the export of the document.
```

Comment:
You can tell that this is a base class (perhaps that should be in the name of the class?) and that it's not intended to be used directly.